src/wallet_cli_main.erl

```diff
@@ -255,6 +271,48 @@ cmd_balance(Opts) ->
                           end, Keys)
     end.
 
+%%
+%% Transactions
+%%
+
+cmd_oui_config(Opts) ->
+    Password = string:strip(string:strip(io:get_line("Passsword: "), right, $\n), right, $\r),
```

Comment:
abstract a get_password/1 which takes a promp as an argument and does this. Replace the other uses of this. Too many repetitions of the same code